contracts/LPVault.sol

```diff
@@ -19,21 +39,21 @@ contract LiquidPledging {
 
 /// @dev `LPVault` is a higher level contract built off of the `Owned`
 ///  contract that holds funds for the liquid pledging system.
-contract LPVault is Owned {
+contract LPVault is Owned {// TODO NEEDS TO BE ESCAPABLE!!! AND WE NEED TO ADD A `WITHDRAW PARTIAL` FUNCTION????????????????
 
-    LiquidPledging public liquidPledging; // liquidPledging contract's address
-    bool public autoPay; // if false, payments will take 2 txs to be completed
+    LiquidPledging public liquidPledging; // LiquidPledging contract's address
+    bool public autoPay; // If false, payments will take 2 txs to be completed
 
     enum PaymentStatus {
-        Pending, // means the payment is awaiting confirmation
-        Paid,    // means the payment has been sent
-        Canceled // means the payment will never be sent
+        Pending, // When the payment is awaiting confirmation
+        Paid,    // When the payment has been sent //TODO CHANGE TO SENT?????????????????????????????????
```

Comment:
either way. I think it's just semantics. They both seem clear to me

Reply:
Its mostly to avoid having 2 similar enum's with `Paid` we could change the other one instead? or if we change the name of the other enum then we are probably fine.